grails-app/services/org/pih/warehouse/invoice/InvoiceService.groovy

```diff
@@ -61,32 +62,32 @@ class InvoiceService {
         return invoiceItems
     }
 
-    def getInvoiceCandidates(String id, String orderNumber, String shipmentNumber, String max, String offset) {
+    def getInvoiceCandidates(String id, String orderNumber, String shipmentNumber) {
```

Comment:
@awalkowiak @pmuchowski Not sure where this comment should go, but next time we make changes to the invoice feature please change InvoiceCandidate to InvoiceItemCandidate (view, domain, etc). It's been bothering me since the initial PR, but I didn't want to delay the API any longer. But now that the API is working I'd like to refactor this.

Also, do we need an InvoiceService and InvoiceItemService? I'd prefer if we have one service per feature / component.

Reply:
Ok. I'll do it.